### PR TITLE
Import Data into TiDB Cloud via MySQL CLI

### PR DESCRIPTION
**Pull request by hooopo**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/import-with-mysql-cli.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+# Import MySQL Data via MySQL Commands in Terminal
+
+## Prerequisites:
+
+Before you can import MySQL data to TiDB Cloud, you should have the following prerequisites in place:
+
+- Access to your TiDB Cloud Cluster.
+- Necessary connection details obtained from the TiDB Cloud portal.
+- An initial password created for your TiDB user (if not already done).
+- TiDB cluster connection information.
```
Comment (sykp241095) [inline tidb-cloud/import-with-mysql-cli.md:15]: Can we move the mysql-client installation here as prerequisites?

File: tidb-cloud/import-with-mysql-cli.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+---
+title: Import Data into TiDB Cloud via MySQL Commands in Terminal
+summary: Learn how to import Data into TiDB Cloud via MySQL CLI.
+---
+
+# Import MySQL Data via MySQL Commands in Terminal
+
+## Prerequisites:
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:8]:
```suggestion
## Prerequisites
```

Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:8]:
```suggestion
## Prerequisites
```

File: tidb-cloud/import-with-mysql-cli.md (hunk 1 above)
Comment (Icemap) [reply tidb-cloud/import-with-mysql-cli.md:15]: For this part, we only need our audience to create a `TiDB Cloud Cluster` and install `MySQL Client` beforehand.
You can get some pre-written chunk form:
- TiDB Cloud Cluster create steps: https://github.com/pingcap/docs/blob/master/develop/dev-guide-gui-datagrip.md?plain=1#L21-L43
- MySQL Client installation: https://github.com/pingcap/docs/blob/master/develop/dev-guide-build-cluster-in-cloud.md?plain=1#L62-L131

File: tidb-cloud/import-with-mysql-cli.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- Access to your TiDB Cloud Cluster.
+- Necessary connection details obtained from the TiDB Cloud portal.
+- An initial password created for your TiDB user (if not already done).
+- TiDB cluster connection information.
+
+## Steps:
+
+### 1. Connect to your TiDB Cloud Cluster
+
+1. Access the TiDB Cloud portal and obtain the necessary connection details.
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:21]: Maybe giving the audience instructions on getting connection details is a good idea. Here is an example that has already passed the review: https://github.com/pingcap/docs/blob/master/develop/dev-guide-gui-vscode-sqltools.md?plain=1#L44-L183

File: tidb-cloud/import-with-mysql-cli.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+2. Create an initial password for your TiDB cluster if you haven't already.
+3. Select your connect options: Choose 'Endpoint Type' as 'Public', select 'Connect With' as 'MySQL CLI' and choose the operating system you are using. Obtain the cluster connection information.
+
+### 2. Install MySQL Command Line Client
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:25]: A little bit redundant if we mentioned it in `Prerequisites`.

File: tidb-cloud/import-with-mysql-cli.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+You need to have MySQL Command Line Client installed on your local machine. You can download it from the official MySQL website if it's not already installed.
+
+### 3. Define Table and Insert Sample Data
+
+Before importing data, you need to prepare the table structure and insert real sample data into it. Here's an example using a hypothetical "products" table:
+
+{{< copyable "sql" >}}
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:33]:
```suggestion
```

Comment (Icemap) [reply tidb-cloud/import-with-mysql-cli.md:33]: We don't need this extra tag now.

File: tidb-cloud/import-with-mysql-cli.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+-- Insert sample data into the table
+INSERT INTO products (product_id, product_name, price) VALUES
+    (1, 'Laptop', 999.99),
+    (2, 'Smartphone', 499.99),
+    (3, 'Tablet', 299.99);
+```
+
+### 4. Import Data from SQL File
+
+1. Provide a real SQL file (e.g., `product_data.sql`) that contains the data you want to import. This SQL file should contain INSERT statements with real data.
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:51]: Does the `product_data.sql` refer to the SQL file above? Then we can name this file a little bit earlier.

File: tidb-cloud/import-with-mysql-cli.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+2. Use the following command to import data from the SQL file:
+
+{{< copyable "bash" >}}
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:55]:
```suggestion
```

Comment (Icemap) [reply tidb-cloud/import-with-mysql-cli.md:55]: ditto

File: tidb-cloud/import-with-mysql-cli.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+### 5. Import Data from CSV File
+
+1. Create a database and schema in TiDB to match your data import needs.
+
+2. Provide a sample CSV file (e.g., `product_data.csv`) that contains the data you want to import. Here's an example of what your sample CSV file might look like:
+
+**product_data.csv:**
+
+{{< copyable "csv" >}}
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:70]:
```suggestion
```

File: tidb-cloud/import-with-mysql-cli.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+```csv
+product_id,product_name,price
+4,Laptop,999.99
+5,Smartphone,499.99
+6,Tablet,299.99
+```
+
+3. Use the following command to import data from the CSV file:
+
+    {{< copyable "bash" >}}
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:80]:
```suggestion
```

File: tidb-cloud/import-with-mysql-cli.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+**product_data.csv:**
+
+{{< copyable "csv" >}}
+```csv
+product_id,product_name,price
+4,Laptop,999.99
+5,Smartphone,499.99
+6,Tablet,299.99
+```
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:76]:
```suggestion
    ```csv
    product_id,product_name,price
    4,Laptop,999.99
    5,Smartphone,499.99
    6,Tablet,299.99
    ```
```

File: tidb-cloud/import-with-mysql-cli.md [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+> **Note:**
+> The default database name used here is 'test', and you can either manually create your own database or use the 'create database' command in an SQL file.
+
+### 5. Import Data from CSV File
+
+1. Create a database and schema in TiDB to match your data import needs.
+
+2. Provide a sample CSV file (e.g., `product_data.csv`) that contains the data you want to import. Here's an example of what your sample CSV file might look like:
+
+**product_data.csv:**
```
Comment (Icemap) [inline tidb-cloud/import-with-mysql-cli.md:68]:
```suggestion
    **product_data.csv:**
```

**Comment by hooopo** [comment]
PR to Release 7.1 https://github.com/pingcap/docs/pull/15333

**Pull request closed without merging**